### ISSUE TITLE
Using links to UYD in Wordpress text

### ISSUE THREAD

**Issue by adriatic** [aurelia-ui-toolkits/screenshots#30]
Here is a typical situation, where I am writing an article describing some details about a gallery that is an UYD object:

![image](https://user-images.githubusercontent.com/2712405/58130304-b7a2b180-7be9-11e9-8cb2-cf8d0e729f7c.png)

It would be nice if you can enhance the parsing of the UYD links - at the moment the above depicted link is created as 
```
with Ino's younger sister <a class="UseyourDrive-directlink" href="https://docs.google.com/file/d/19j74wUwaK0PUlQXoevQETfG6gvtOfM4J/view" target="_blank" rel="noopener noreferrer">img002.jpg</a>
```

I am interpreting the current situation as

```
href="https://docs.google.com/file/d/19j74wUwaK0PUlQXoevQETfG6gvtOfM4J/view
```

being the reference to the folder. The reference to a particular file (`img002.jpg` in this case) is built from two parts - path to the folder and the file name.

The unfortunate consequence of this decision that I cannot use it in a fashion that is more standard - and it would allow me to show the image title as a more meaningful text than `img002.jpg`.

In summary, I would like this format

```
with <a class="UseyourDrive-directlink" href="https://docs.google.com/file/d/19j74wUwaK0PUlQXoevQETfG6gvtOfM4J/view/img002.jpg" target="_blank" rel="noopener noreferrer">Ino's younger sister/a>
```

This would allow me to compose the same text section in a more natural way:

![image](https://user-images.githubusercontent.com/2712405/58133017-c6d92d80-7bf0-11e9-85c1-9c3a7a7c9968.png)

where the Lada's image is indeed a link to that `img002,jpg` file, so click on that (small) image results with the full image shown in the new browser window.
